src/jenkins.ts

```diff
@@ -0,0 +1,202 @@
+import { ApiObject, JsonPatch } from 'cdk8s';
+import { Construct } from 'constructs';
+import {
+  Jenkins as JenkinsCustomResource,
+  JenkinsSpecMasterContainers as JenkinsContainerConfiguration,
+  JenkinsSpecMasterContainersResourcesLimits as JenkinsContainerLimits,
+} from './imports/jenkins.io';
+
+export interface Plugins {
+  /**
+   * The url from where plugin has to be downloaded.
+   */
+  readonly downloadUrl?: string;
+
+  /**
+   * The name of Jenkins plugin.
+   */
+  readonly name: string;
+
+  /**
+   * The version of Jenkins plugin.
+   */
+  readonly version: string;
+}
+
+export interface SeedJobs {
```

Comment:
Does this interface represent a collection of seed jobs, or a single seed job?

Reply:
Just one single seed job. Will update this to `SeedJob`.